Issue importing JSON to vaultwarden web

Imported .JSON file that was exported from my bitwarden for windows into vaultwarden web, 

Expected Result: a successful import

WHAT ACTUALLY HAPPENS ERROR: e.match was null (or == null idr)

Must be a bug as I exported .csv from bitwarden for windows then imported and it worked fine.

Thanks <3(i just cloned the repos now so must be newest version)

This is what I get when I try to import my encrypted json to an organization vault:
`vaultwarden  | [2023-10-27 12:19:37.398][vaultwarden::api::core::ciphers][ERROR] The client copy of this cipher is out of date. Resync the client and try again.`
Dozens ,if not hundreds, of messages like this ^^

--- a/src/api/core/ciphers.rs
+++ b/src/api/core/ciphers.rs
@@ -359,14 +359,17 @@ pub async fn update_cipher_from_data(
     enforce_personal_ownership_policy(Some(&data), headers, conn).await?;
 
     // Check that the client isn't updating an existing cipher with stale data.
-    if let Some(dt) = data.LastKnownRevisionDate {
-        match NaiveDateTime::parse_from_str(&dt, "%+") {
-            // ISO 8601 format
-            Err(err) => warn!("Error parsing LastKnownRevisionDate '{}': {}", dt, err),
-            Ok(dt) if cipher.updated_at.signed_duration_since(dt).num_seconds() > 1 => {
-                err!("The client copy of this cipher is out of date. Resync the client and try again.")
+    // And only perform this check when not importing ciphers, else the date/time check will fail.
+    if ut != UpdateType::None {
+        if let Some(dt) = data.LastKnownRevisionDate {
+            match NaiveDateTime::parse_from_str(&dt, "%+") {
+                // ISO 8601 format
+                Err(err) => warn!("Error parsing LastKnownRevisionDate '{}': {}", dt, err),
+                Ok(dt) if cipher.updated_at.signed_duration_since(dt).num_seconds() > 1 => {
+                    err!("The client copy of this cipher is out of date. Resync the client and try again.")
+                }
+                Ok(_) => (),
             }
-            Ok(_) => (),
         }
     }
 

